src/App.tsx

```diff
@@ -1,7 +1,39 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    key: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener(
+      'keyup', (event: KeyboardEvent) => {
+        this.setState({ key: event.key });
+      },
```

Comment:
please create separate, resusable function for this logic - both functions are doing the same thing